index.d.ts

```diff
@@ -178,6 +178,9 @@ declare namespace suitest {
 			status: string,
 			displayName?: string,
 			shortDisplayName?: string,
+			customUserInfo?: {
+				otherInfo: string,
```

Comment:
Hi, thanks a lot for your contribution!

The current type definition is not entirely correct and still misses a few fields. For example, the `otherInfo` field should be defined as optional (`otherInfo?: string`) rather than required.

You can find the complete reference here:
https://suite.st/docs/suitest-network-api/api-reference/
`GET /devices/{deviceId}`

If you'd like, it would be great to add the missing fields as well.

Thanks again for your effort!